lib/muffin_man/notifications/v1.rb

```diff
@@ -0,0 +1,63 @@
+module MuffinMan
+  module Notifications
+    require "json"
+    require "sp_api_helpers"
+    class V1 < SpApiClient
+
+      NOTIFICATION_PATH = "/notifications/v1"
+
+      def create_destinations(arn, params={})
+        params = params.transform_keys{|key| key.to_s}
+        @local_var_path = "#{NOTIFICATION_PATH}/destinations"
+        destination_params = {"resourceSpecification": {"sqs": {"arn": arn}}, "marketplaceIds": params["marketplace_ids"]}
+        destination_params.merge!("eventBridge": {"region": params["region"], "accountId": params["account_id"]}) unless params["region"].nil? || params["account_id"].nil?
+        destination_params.merge!("name": params["name"]) unless params["name"].nil?
+        @request_body = destination_params
+        @request_type = "POST"
+        call_api
+      end
+
+       def get_destinations(params={})
```

Comment:
This is maybe strange, but it seems like there are no parameters you can pass for this endpoint, so I'd just remove the params option here

Reply:
I have kept it purposefully for future scope, as it returns collection of data so some filter parameters might be required. It is not harmful as I've set it as default blank.